plugins/modules/pagerduty_alert.py

```diff
@@ -111,11 +112,12 @@
     type: str
     description:
       - Identifies the incident to which this O(state) should be applied.
-      - For O(state=triggered) - If there is no open (in other words unresolved) incident with this key, a new one will be created. If there is already an
-        open incident with a matching key, this event will be appended to that incident's log. The event key provides an easy way to 'de-dup'
-        problem reports. If no O(incident_key) is provided, then it will be generated by PagerDuty.
-      - For O(state=acknowledged) or O(state=resolved) - This should be the incident_key you received back when the incident was first opened
-        by a trigger event. Acknowledge events referencing resolved or nonexistent incidents will be discarded.
+      - For O(state=triggered) - If there is no open (in other words unresolved) incident with this key, a new one will be
+        created. If there is already an open incident with a matching key, this event will be appended to that incident's
+        log. The event key provides an easy way to 'de-dup' problem reports. If no O(incident_key) is provided, then it will
+        be generated by PagerDuty.
+      - For O(state=acknowledged) or O(state=resolved) - This should be the incident_key you received back when the incident
```

Comment:
```suggestion
      - For O(state=acknowledged) or O(state=resolved) - This should be the RV(ignore:incident_key) you received back when the incident
```

Reply:
I believe that would make sense, but the RETURN values are not documented. @ApsOps @xshen1 could you please confirm? TIA